src/gluonnlp/data/tokenizers/sentencepiece.py

```diff
@@ -131,12 +131,11 @@ def __init__(self, model_path: Optional[str] = None,
         else:
             self._vocab = load_vocab(vocab)
         # Sanity check
-        assert self._vocab.all_tokens == sp_model_all_tokens
+        assert len(self._vocab.all_tokens) >= len(sp_model_all_tokens)
+        assert self._vocab.all_tokens[:len(sp_model_all_tokens)] == sp_model_all_tokens
         for token in self._vocab.special_tokens:
             piece_id = self._sp_model.piece_to_id(token)
-            if self._sp_model.is_unknown(piece_id):
-                assert self._vocab[token] == self._sp_model.unk_id()
```

Comment:
Does the original T5 model contains a unk_id?

Reply:
If so, we may still keep this.

Reply:
In this PR, `gluon._sp_model.unk_id()` still corresponds to `<unk>`, as desired. The reason for deleting line 137 and 138 is that `<extra_id_{0..99}>` also satisfies this clause, but the subsequent `assert` will fail.